mockgcp/common/httpmux/json.go

```diff
@@ -0,0 +1,59 @@
+package httpmux
```

Comment:
Header? copyright et al

Reply:
Good catch ... hopefully our tests will also yell at me about that!